analysis/src/Xform.ml

```diff
@@ -310,5 +462,7 @@ let extractCodeActions ~path ~pos ~currentFile ~debug =
     AddTypeAnnotation.xform ~path ~pos ~full ~structure ~codeActions ~debug;
     IfThenElse.xform ~pos ~codeActions ~printExpr ~path structure;
     AddBracesToFn.xform ~pos ~codeActions ~path ~printStructureItem structure;
+    TypeToModule.xform ~path ~pos ~codeActions ~printStructureItem structure
```

Comment:
We have quite a few of these.
Need a little sanity check:
- that they don't do repeat work in a way that hurts perf (iterating several times is probably ok while copying would not)
- that no work is done unless the action fires

This is just about reviewing the code and check that things are OK.